assets/css/accredible-admin-settings.css

```diff
@@ -290,6 +291,11 @@ button[class*="accredible-button"] {
     max-width: 500px;
 }
 
+#issuer-info {
```

Comment:
Subjective - This should not cause any problems but I'd add the `accredible-` prefix for consistency and just in case.